dotCMS/src/main/java/com/dotcms/rest/api/v1/template/TemplateResource.java

```diff
@@ -159,8 +160,9 @@ public final Response list(@Context final HttpServletRequest httpRequest,
         final InitDataObject initData = new WebResource.InitBuilder(webResource)
                 .requestAndResponse(httpRequest, httpResponse).rejectWhenNoUser(true).init();
         final User user = initData.getUser();
+        final Lazy<String> lazyCurrentHost = Lazy.of(() -> Try.of(() -> Host.class.cast(httpRequest.getSession().getAttribute(WebKeys.CURRENT_HOST)).getIdentifier()).getOrNull());
```

Comment:
Is it worth to add a test for the case when `lazyCurrentHost` is null?

Reply:
There is already a postman test where check this case, no current host set neither the query param.